src/core/services/interaction/interaction.spec.js

```diff
@@ -0,0 +1,26 @@
+describe("$mdInteraction", function() {
```

Comment:
I'd love to see some additional tests here. Since the focus style hinges off of `.md-focused` being added, we could test that it gets added on keyboard interaction and not mouse.